src/main/webapp/app/shared/model/field-type.model.ts

```diff
@@ -0,0 +1,56 @@
+import { Constraint } from './field.model';
+import { PropertyType } from './property.model';
+
+export interface FieldType {
```

Comment:
when you have a field type as model...it should have that model only..this file contains other models and enum. create a new file for each model and put enums in constants

Reply:
done